### ISSUE TITLE
duplicate control in xml view

### ISSUE THREAD

**Issue by choper725** [abap2UI5/abap2UI5#491]
we need some general guide lines from inserting new control to xml view..

it starting to build up duplicate control..
some with underscore in the name and some without..

**Comment by oblomov-dev** [comment]
generally i think the best guide line is the following:

Method name = Control Name but in snake case (cleaned it up now, there were a few methods more than one time )
importing parameter = propertyname (exactly as it is in the ui5 documentation and typed with clike and optional)

Or any other suggestions?

Will add this to the documentation.

**Comment by choper725** [comment]
i agree with you, it should not be more complicated then that..

just to keep it tidy, of course any pr is great, just to try not making it a mess, because if duplicates stay, and people will start using this project in prod, this can lead to alot of unessecary CR work :)

**Comment by oblomov-dev** [comment]
yes you are right, will try to keep an eye on this when checking incoming PRs
